src/sql_utils.h

```diff
@@ -0,0 +1,123 @@
+#ifndef SQL_UTILS_H_
+#define SQL_UTILS_H_
+
+#include <memory>
+
```

Comment:
Could you #include "logging.h" here?  It will at least make my IDE happier :)

Reply:
your ide is right :), fixed